rally_ovs/plugins/ovs/deployment/engines/ovs/ovs-sandbox.sh

```diff
@@ -694,19 +735,35 @@ EOF
 
     # Initialize database.
     if $controller ; then
-        init_ovsdb_server "ovsdb-server-nb" $OVN_NB_DB
-        init_ovsdb_server "ovsdb-server-sb" $OVN_SB_DB
+        if $enable_ssl ; then
+            tar -xzvf certs.tar.gz
+            abs_path=`pwd`/certs
+            ovn-nbctl set-ssl $abs_path/ovnnb-privkey.pem  $abs_path/ovnnb-cert.pem \
+                $abs_path/cacert.pem
+            ovn-nbctl set-connection pssl:6641:127.0.0.1
```

Comment:
why 127.0.0.1? Rally may need to access NB remotely.

Reply:
So by default it will be localhost where we are setting up controller sandbox. We are not using cluster here. Hence when we run all the nbctl commands too , we source sandbox rc and do it. Once we introduce to spin up clustered db, we can leverage the actual IPs. Not needed now.